microservices/Api.Pdp/Properties/launchSettings.json

```diff
@@ -30,17 +30,17 @@
       "commandName": "Project",
       "launchBrowser": true,
       "launchUrl": "values",
-      "applicationUrl": "https://localhost:5003;http://localhost:5002",
       "environmentVariables": {
         "ASPNETCORE_ENVIRONMENT": "Development"
-      }
+      },
+      "applicationUrl": "https://localhost:5003;http://localhost:5002"
     },
-      "Docker": {
-        "commandName": "Docker",
-        "launchBrowser": true,
-        "launchUrl": "{Scheme}://{ServiceHost}:{ServicePort}/values",
-        "publishAllPorts": true,
-        "useSSL": true
-      }
+    "Docker": {
+      "commandName": "Docker",
+      "launchBrowser": true,
+      "launchUrl": "{Scheme}://{ServiceHost}:{ServicePort}/values",
```

Comment:
Should it be the same as the launchUrl?

Reply:
nah, this is for actually starting it up when running deployed, there we want it to just start at values i guess